importer.py

```diff
@@ -21,4 +21,37 @@
                              attachment_key='org.digitalpsych.redcap.data',
                              body=records)
     print(count)
+    try:
+        arr = LAMP.Type.get_attachment(researcher['id'], 'org.digitalpsych.redcap.links')["data"]
+    except LAMP.ApiException:
+        continue # Ignore this Researcher
+    parts = []
+    for study in LAMP.Study.all_by_researcher(researcher['id'])['data']:
+        parts+=(p['id'] for p in LAMP.Participant.all_by_study(study['id'])['data'])
+    for p in parts:
+        try:
+            record_id = LAMP.Type.get_attachment(p, 'org.digitalpsych.redcap.id')["data"]
```

Comment:
Nice! 👍 It's good API design that the `org.digitalpsych.redcap.id` Tag is on **each** Participant. Maybe in the future we could add flags to import things in different ways per-patient.